server/routes/carts.js

```diff
@@ -0,0 +1,113 @@
+const Router = require('express')
+const Cart = require('../models/Cart')
+const router = new Router()
+
+router.post('/api/carts', createCart)
```

Comment:
This method shouldn't be opened for all users without `auth`, if some user could read all available carts, he can add pizzas to all cards, just for fun

<img width="924" alt="image" src="https://user-images.githubusercontent.com/12561302/116731535-184be300-a9ea-11eb-9403-91824b8e853f.png">